shared/constants/maps.ts

```diff
@@ -82,7 +82,7 @@ export const mapConfig: MapConfigProps = {
 export const defaultViewportConfig: MapViewViewport = {
   latitude: 48.366512,
   longitude: 10.894446,
-  zoom: 8,
+  zoom: 10,
```

Comment:
🤔 Why did you change this?

Reply:
Came with the boundingBox changes, I'll revert it